tutorial/source/baseball.rst

```diff
@@ -0,0 +1,11 @@
+Example: analyzing baseball stats with MCMC
+=======================================================
```

Comment:
don't these lengths need to match exactly? not sure how this markdown works...

Reply:
I find `rst` confusing but it seems to render correctly for me locally.

Reply:
The bars need to be *at least* as long as the text.